src/api/files.rs

```diff
@@ -0,0 +1,49 @@
+//!
+//! Support for Slack Files API methods
+//!
+
+use rsb_derive::Builder;
+use serde::{Deserialize, Serialize};
+use serde_with::skip_serializing_none;
+
+use crate::models::*;
+use crate::ratectl::*;
+use crate::SlackClientSession;
+use crate::{ClientResult, SlackClientHttpConnector};
+
+impl<'a, SCHC> SlackClientSession<'a, SCHC>
+where
+    SCHC: SlackClientHttpConnector + Send,
+{
+    ///
+    /// https://api.slack.com/methods/files.upload
+    ///
+    pub async fn files_upload(
+        &self,
+        req: &SlackApiFilesUploadRequest,
+    ) -> ClientResult<SlackApiFilesUploadResponse> {
+        self.http_session_api
+            .http_post_form_urlencoded("files.upload", req, Some(&SLACK_TIER2_METHOD_CONFIG))
+            .await
+    }
+}
+
+#[skip_serializing_none]
+#[derive(Debug, PartialEq, Clone, Serialize, Deserialize, Builder)]
+pub struct SlackApiFilesUploadRequest {
+    pub channels: SlackChannelId,
```

Comment:
`channels` in plural here looks suspicious since the type accepts only one here.

Reply:
Ok, checked the API docs.
This should `Vec<SlackChannelId>` and we need to make comma separated string from it.